app-server/datamodel/src/main/scala/com/azavea/rf/datamodel/latest/schema/schema.scala

```diff
@@ -212,6 +212,91 @@ trait tables {
   /** Collection-like TableQuery object for table ScenesToBuckets */
   lazy val ScenesToBuckets = new TableQuery(tag => new ScenesToBuckets(tag))
 
+  /** Entity class storing rows of table Thumbnails
+   *  @param id Database column id SqlType(uuid), PrimaryKey
+   *  @param createdAt Database column created_at SqlType(timestamp)
+   *  @param modifiedAt Database column modified_at SqlType(timestamp)
+   *  @param organizationId Database column organization_id SqlType(uuid)
+   *  @param createdBy Database column created_by SqlType(varchar), Length(255,true)
+   *  @param widthPx Database column width_px SqlType(int4)
+   *  @param heightPx Database column height_px SqlType(int4)
+   *  @param size Database column size SqlType(size)
+   *  @param scene Database column scene SqlType(uuid), Default(None)
+   *  @param url Database column url SqlType(varchar), Length(255,true) */
+  case class ThumbnailsRow(id: java.util.UUID, createdAt: java.sql.Timestamp, modifiedAt: java.sql.Timestamp, organizationId: java.util.UUID, createdBy: String, widthPx: Int, heightPx: Int, size: String, scene: Option[java.util.UUID] = None, url: String)
```

Comment:
It doesn't fail to compile because the `schema` represents what _should_ be in the database -- there's not really a way for the compiler to know what is or isn't in the database.

It would cause a runtime exception if at some point this table was interacted with.

Once you change the `sql` migration, I think you need to do the following to keep things in sync (in development):
 - delete the table/revert any changes from the migration manually (can also just drop the database and start from scratch)
 - delete any generated code from that migration - e.g. the latest `schema.scala` and the one generated for this particular migration
 - if you don't drop the whole database, delete the migration from `__migrations__` in the database

It's kind of tedious, dropping the whole database is the path of least resistance at this point since we don't have much in there anyways though.